symengine/dense_matrix.cpp

```diff
@@ -175,14 +181,75 @@ void jacobian(const DenseMatrix &A, const DenseMatrix &x, DenseMatrix &result)
     SYMENGINE_ASSERT(A.col_ == 1);
     SYMENGINE_ASSERT(x.col_ == 1);
     SYMENGINE_ASSERT(A.row_ == result.nrows() and x.row_ == result.ncols());
+    bool error = false;
+#pragma omp parallel for
     for (unsigned i = 0; i < result.row_; i++) {
         for (unsigned j = 0; j < result.col_; j++) {
             if (is_a<Symbol>(*(x.m_[j]))) {
                 const RCP<const Symbol> x_
                     = rcp_static_cast<const Symbol>(x.m_[j]);
                 result.m_[i * result.col_ + j] = A.m_[i]->diff(x_);
             } else {
-                throw std::runtime_error("'x' must contain Symbols only");
+                error = true;
+                break;
+            }
+        }
+    }
+    if (error) {
+        throw std::runtime_error("'x' must contain Symbols only. "
+            "Use sjacobian for SymPy style differentiation");
+    }
+}
+
+void sjacobian(const DenseMatrix &A, const DenseMatrix &x, DenseMatrix &result)
+{
+    SYMENGINE_ASSERT(A.col_ == 1);
+    SYMENGINE_ASSERT(x.col_ == 1);
+    SYMENGINE_ASSERT(A.row_ == result.nrows() and x.row_ == result.ncols());
+#pragma omp parallel for
```

Comment:
Only when SymEngine is compiled with `WITH_OPENMP` and the compiler supports openmp standard